helm/etcd-kubernetes-resources-count-exporter/values.yaml

```diff
@@ -56,3 +56,7 @@ podSecurityContext:
 securityContext:
   seccompProfile:
     type: RuntimeDefault
+
+kyvernoPolicyExceptions:
```

Comment:
Wouldn't this make the app not work in legacy clusters?

Reply:
With the [capability gate](https://github.com/giantswarm/etcd-kubernetes-resources-count-exporter/pull/64/files#diff-d45ffff7f0d3917926869f5fa5db1614e826d86d5d57e819a06217aa9d0fa526R2), the polex wouldn't be created in clusters without Kyverno.

My slight concern is for cases where this is installed into a cluster without Kyverno and then Kyverno is later installed. The PolicyException would not be created until this app is deployed again, so there is some time where the pods could be rejected if policies are being enforced. This is a general problem Shield is working on so I would suggest this PR can continue and we'll keep working on a more general solution.